transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollStreamChannel.java

```diff
@@ -824,7 +824,7 @@ void epollInReady() {
                     allocHandle.incMessagesRead(1);
                     pipeline.fireChannelRead(byteBuf);
                     byteBuf = null;
-                } while (allocHandle.continueReading());
+                } while (config.isAutoRead() && allocHandle.continueReading());
```

Comment:
I'm glad you picked this one out. I do want to change this, but I'm not sure what I need to do... There are a few implementations of ```RecvByteBufAllocator``` and I'd like @normanmaurer to give me some guidance on how to include it correctly.